src/sfeTk/sfeTkII2C.h

```diff
@@ -34,7 +34,7 @@ class sfeTkII2C : public sfeTkIBus
     sfeTkII2C() : _address{kNoAddress}
     {
     }
-    sfeTkII2C(uint8_t addr) : _address{addr}
+    sfeTkII2C(uint8_t addr) : _address{addr}, _stop{true}
```

Comment:
I'd move the initialization of _stop{true} to the default ctor - the constructor without arguments(the method above this one). 

And since _stop is a uint8_t, should the default value be 1? I would specify the number, not the value of ```true``` in the initializer).

Reply:
Ah, yes. 1 is the correct choice. The Arduino source uses a mixture of uint8_t but with bool terminology. Let's stick with 1.